monai/networks/nets/basic_unetplusplus.py

```diff
@@ -0,0 +1,317 @@
+# Copyright (c) MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Optional, Sequence, Union
+
+import torch
+import torch.nn as nn
+
+from monai.networks.blocks import Convolution, UpSample
+from monai.networks.layers.factories import Conv, Pool
+from monai.utils import ensure_tuple_rep
+
+__all__ = ["BasicUnetPlusPlus", "BasicunetPlusPlus", "basicunetplusplus", "BasicUNetPlusPlus"]
+
+
+class TwoConv(nn.Sequential):
+    """two convolutions."""
+
+    def __init__(
+        self,
+        spatial_dims: int,
+        in_chns: int,
+        out_chns: int,
```

Comment:
We've used these keywords elsewhere in MONAI, we should try to have a standard though I know many places have different names too:
```suggestion
        in_channels: int,
        out_channels: int,
```

Reply:
Alright @ericspod, I'll update these

Reply:
@ericspod, I've updated the variable names.

Reply:
Don't know why, but updating just the variable names is causing Flake8 issues.

Reply:
It's black complaining about the formatting it looks like, perhaps lines are too long now, you can get black to fix things with `./runtests.sh --autofix` or the `/black` action command.